### PR TITLE
Copy to clipboard

### LINKED ISSUE #4579: text not saved of `Copy to clipboard` action for `on click` event of table's action button
### Is your proposal related to a problem?

When adding the `on click` event for the action button on the table widget and selecting the `Copy to Clipboard` action, when you enter the text, it won't save. It resets after you close the event handler popup.

### Describe the solution you'd like

Fix the `Copy to clipboard` action for the action button

[copy-to-clipboard.webm](https://user-images.githubusercontent.com/25361949/197396927-f2271097-0ed2-4946-821e-153519698875.webm)

### PR DESCRIPTION
**Pull request by Smartmind12**
Closes #4579

### CODE REVIEW

**Comment by gsmithun4** [comment]
deployed https://tj-4588.herokuapp.com

File: frontend/src/Editor/Inspector/EventManager.jsx [hunk 1]
```diff
@@ -343,7 +343,8 @@ export const EventManager = ({
                 <CodeHinter
                   theme={darkMode ? 'monokai' : 'default'}
                   currentState={currentState}
-                  onChange={(value) => handlerChanged(index, 'contentToCopy', value)}
+                  initialValue={event.text}
+                  onChange={(value) => handlerChanged(index, 'text', value)}
```
Comment (kavinvenkatachalam) [inline frontend/src/Editor/Inspector/EventManager.jsx:347]:
```suggestion
                  initialValue={event.contentToCopy}
                  onChange={(value) => handlerChanged(index, 'contentToCopy', value)}
```

**Review by kavinvenkatachalam — CHANGES_REQUESTED** [review]
@Smartmind12 Thanks for the PR.

Seems like while clicking the button, the value entered is not copying to the clipboard. Suggested a change below, please check and update the code

File: frontend/src/Editor/Inspector/EventManager.jsx (hunk 1 above)
Comment (Smartmind12) [reply frontend/src/Editor/Inspector/EventManager.jsx]: @kavinvenkatachalam I have made the changes...so it should work now ...kindly review!

**Review by kavinvenkatachalam — APPROVED** [review]
LGTM

**Comment by withshubh** [comment]
Hey @Smartmind12 
Thanks for contributing to ToolJet. 🙌 
Please submit the form to claim your Hacktoberfest swag: https://airtable.com/shr1qR6xhqsWO7LD7

**Pull request merged**